lib/python/picongpu/plugins/data/compare.py

```diff
@@ -0,0 +1,222 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2017-2018 PIConGPU contributors
+Authors: Sophie Rudat
+License: GPLv3+
+"""
+from picongpu.plugins.emittance import EmittanceData
+from picongpu.plugins.energy_histogram import EnergyHistogramData
+
+import numpy as np
+import os
+import collections
+import json
+
+
+class CompareData(object):
+    """
+    Data Reader for the Energy Histogram Plugin.
+    """
+
+    def __init__(self, run_directory):
+        """
+        Parameters
+        ----------
+        simulation_directory : string
+            path to the run directory of PIConGPU
+            (the path before ``simOutput/``)
+        """
+        if run_directory is None:
```

Comment:
this check is already done in the base class, so you can omit it when you call the base class constructor